src/amo/containers/Home.js

```diff
@@ -6,6 +7,22 @@ import translate from 'core/i18n/translate';
 
 import 'amo/css/Home.scss';
 
+const CategoryLink = ({ name, slug, text, type }) => (
```

Comment:
To be clear I think the `ThemeLink` and `ExtensionLink` components are fair (and could even be exported from `CategoryLink` but they should be in their own file.